.github/workflows/create-release.yml

```diff
@@ -1,96 +1,78 @@
-name: "Create release"
+name: Create release
 
 on:
   workflow_dispatch:
+    inputs:
+      DRY_RUN:
+        type: boolean
+        default: false
+        description: "Dry run: setting this to `true` will skip the final step of publishing the release, leaving it as a draft-release. This can be helpful for testing and debugging. Note that the tag and release are still getting generated and the next run of this result in incremented release version. To avoid this, do not forget to delete the resulting release and tag."
 
 jobs:
-  verify-release:
-    name: Verify release
-    runs-on: ubuntu-latest
-    outputs:
-      version: ${{ steps.gen-version.outputs.VERSION }}
+  gen-version:
+    name: Generate semantic version from branch and tags
+    uses: kyma-project/eventing-tools/.github/workflows/get-version-from-release-branch-reusable.yml@main
```

Comment:
Will this always return the workflow from main~head? If yes, can we use a commit SHA instead, just in case main~head get broken for some reason.

Reply:
Well, I am happy that you ask about this because I was also between these two options.
Sticking this to a commit sha makes this less susceptible to bug, so yes, let's do that.

Reply:
I changed it to a commit sha everywhere.

Reply:
I need to update the `create-draft-release` workflow sha, after [this PR](https://github.com/kyma-project/eventing-tools/pull/66) got merged.